main.py

```diff
@@ -18,24 +19,25 @@ def signup():
     password = request.form["password"]
     verify = request.form["verify"]
     email = request.form["email"]
-    string_len = True
+    
+    username_error = ""
+    password_error = ""
+    verify_error = ""
+    email_error = ""
```

Comment:
Grouped all the default values into one place, this way we don't need to use any `else` statements.